p2p/conn/connection.go

```diff
@@ -856,7 +861,8 @@ func (ch *Channel) writePacketMsgTo(w io.Writer) (n int, err error) {
 		err = ErrPacketWrite{Source: err}
 	}
 
-	atomic.AddInt64(&ch.recentlySent, int64(n))
+	// increment recentlySent by 1, to denote we sent another message.
```

Comment:
But this is not a message, but a packet. A message is composed of multiple packets.